File: php/GestionAbsence/classes/Retard.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Retard
 *
 * @author Abdelilah Boutizoua
 */
class Retard {
    
    private $id;
    private $stagiaire;
    private $matiere;
    private $dateRetard;
    private $nbre;

    function __construct($stagiaire, $matiere, $dateRetard,$nbre) {
        $this->stagiaire = $stagiaire;
        $this->matiere = $matiere;
        $this->dateRetard = $dateRetard;
        $this->nbre = $nbre;
    }
    
    function getId() {
        return $this->id;
    }

    function getStagiaire() {
        return $this->stagiaire;
    }

    function getMatiere() {
        return $this->matiere;
    }

    function getDateRetard() {
        return $this->dateRetard;
    }


    function getNbreHeure() {
        return $this->nbre;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setStagiaire($stagiaire) {
        $this->stagiaire = $stagiaire;
    }

    function setMatiere($matiere) {
        $this->matiere = $matiere;
    }

    function setDateRetard($dateRetard) {
        $this->dateRetard = $dateRetard;
    }

}

File: app/gestiondabsence/app/src/main/java/com/example/dell/gestiondabsence/Bean/Retard.java
package com.example.dell.gestiondabsence.Bean;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Abdelilah Boutizoua on 12/04/2018.
 */

public class Retard {

    private int id;

    @SerializedName("stagiaire_id")
    private int stagiaire;

    @SerializedName("matiere_id")
    private int matiere;

    @SerializedName("date_retard")
    private String dateRetard;

    @SerializedName("nbre_heure")
    private int nbre_heure;

    public Retard(int stagiaire, int matiere, String dateRetard, int nbre_heure) {
        this.stagiaire = stagiaire;
        this.matiere = matiere;
        this.dateRetard = dateRetard;
        this.nbre_heure = nbre_heure;
    }

    public int getId() {
        return id;
    }

    public int getStagiaire() {
        return stagiaire;
    }

    public int getMatiere() {
        return matiere;
    }

    public String getDateRetard() {
        return dateRetard;
    }
}
